Changelog.md

```diff
@@ -8,6 +8,7 @@ __New Features__
   - Adds "Peak Electricity: Annual Total (W)" output.
   - Adds battery resilience hours output; allows requesting timeseries output.
   - ReportUtilityBills measure: Allows reporting monthly utility bills in addition to (or instead of) annual bills.
+- Update to 2022 EIA energy costs.
```

Comment:
In the documentation, there are two references to this EIA data with respect to defaulting. Can you add "2022" to their description?